lesson13/homework_13_xml.py

```diff
@@ -0,0 +1,46 @@
+# Завдання 3:
+# Для файла ideas_for_test/work_with_xml/groups.xml створіть функцію пошуку по group/number і
+# повернення значення timingExbytes/incoming результат виведіть у консоль через логер на рівні інфо
+
+import xml.etree.ElementTree as ET
+import logging
+from pathlib import Path
+
+
+logging.basicConfig(
+    level=logging.INFO,
+    format='%(asctime)s - %(levelname)s - %(message)s'
+)
+
+folder_path = Path('C:\\Users\\007\\Downloads')
```

Comment:
It is better to store project files inside Automation project repository. In this case no need to have unique paths.